packages/examples/create-react-app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
hm perhaps the example app should live in a different repo? it might get a bit crowded in here once we land examples for various frameworks

Reply:
yeah, I agree it should, was just really for testing things whilst we are still figuring out the apis etc